weave-js/src/components/Panel2/PanelPlot/PanelPlot.tsx

```diff
@@ -423,62 +92,6 @@ const PanelPlotConfigInner: React.FC<PanelPlotProps> = props => {
     updateConfig(newConfig);
   }, [weave, config, updateConfig]);
 
-  // const exportAsCode = useCallback(() => {
-  //   if (navigator?.clipboard == null) {
-  //     return;
-  //   }
-
-  //   if (config.series.length !== 1) {
-  //     toast('Multi-series plots are not currently supported');
-  //     return;
-  //   }
-
-  //   const series = config.series[0];
-
-  //   const dimConfigured = (dim: keyof SeriesConfig['dims']) =>
-  //     series.table.columnSelectFunctions[series.dims[dim]].type !== 'invalid';
-
-  //   if (!dimConfigured('x') || !dimConfigured('y')) {
-  //     toast(
-  //       "Can't export to code: Required dimensions x and/or y are not configured"
-  //     );
-  //     return;
-  //   }
-
-  //   const dimArgument = (dim: keyof SeriesConfig['dims']) =>
-  //     `${dim}=lambda row: ${weave.expToString(
-  //       series.table.columnSelectFunctions[series.dims[dim]],
-  //       null
-  //     )},`;
-
-  //   const inputTypeText = toPythonTyping(input.type);
-
-  //   const dims: Array<keyof SeriesConfig['dims']> = [
-  //     'x',
-  //     'y',
-  //     'label',
-  //     'tooltip',
-  //   ];
-
-  //   const codeText = [
-  //     '@weave.op()',
-  //     `def my_panel(input: weave.Node[${inputTypeText}]) -> panels.Plot:`,
-  //     '  return panels.Plot(',
-  //     '    input,',
-  //     ...dims.reduce<string[]>((memo, field) => {
-  //       if (dimConfigured(field)) {
-  //         memo.push(`    ${dimArgument(field)}`);
-  //       }
-  //       return memo;
-  //     }, [] as string[]),
-  //     '  )\n',
-  //   ].join('\n');
-
-  //   navigator.clipboard
-  //     .writeText(codeText)
-  //     .then(() => toast('Code copied to clipboard!'));
-  // }, [config, input.type, weave]);
```

Comment:
We should keep this in a commented out form in case we want to re-enable it later on